Examples/MAX32660/UART/main.c

```diff
@@ -194,16 +194,10 @@ int main(void)
     if (fail != 0) {
         LED_On(0);
         printf("-->EXAMPLE FAILED\n");
+        return E_FAIL;
     } else {
         printf("-->EXAMPLE SUCCEEDED\n");
     }
 
-    while (1) {
```

Comment:
The PR shall not change example functionality (as possible as),
In this case at the end of example led blinks, instead of removing this feature you should add a counter to blink the led n times, n can be 10.

Reply:
@ozersa which line blinks the LED? I am seeing that it is either off or on. Since GPIO's will preserve the state after code finishes, it should work as before. Am I wrong?

Reply:
See below.
![image](https://user-images.githubusercontent.com/46590392/211260123-98b906f6-3086-4ea5-9fff-e6ef8ad8c10e.png)

Reply:
It is okay that we have such a while loop at the end. However, I am checking the code and it initially has blocking empty while loops at the top that changed to a simple return. I mean this portion mostly is not executed. Moreover, the other UART examples do not have such a loop in their original form. Therefore, removing this portion makes this example similar to the other UART examples.
Do we really need such a portion? Why others do not have it but MAX32660 has? @ozersa